### PR TITLE
Closes #1097: added requestee url validation

### LINKED ISSUE #1097: FE: Validation for requestee in URL (S)
## Description
The `for` parameter in the URL contains the member ID of the requestee. There need to be additional validation checks to ensure this is present throughout the request feedback stepper.

## Definition of Done
- [x] User cannot navigate to `/feedback/request` without specifying a valid `for` parameter
- [x] Next button is disabled if there is no requestee in the URL

Depends on issue #1057 and issue #1054

### PR DESCRIPTION
**Pull request by Approve212**

### CODE REVIEW

**Review by Maroju100 — APPROVED** [review]

**Review by moshirim — APPROVED** [review]

**Review by zsbrown97 — APPROVED** [review]

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 1]
```diff
@@ -82,29 +65,20 @@ const FeedbackRequestPage = () => {
...
-  useEffect(() => {
-    async function getMemberProfile() {
-      if (id) {
-        let res = await getMember(id, csrf);
-        let requesteeProfile =
-          res.payload.data && !res.error ? res.payload.data : undefined;
-        setRequestee(requesteeProfile ? requesteeProfile.name : "");
-      }
-    }
+  const [preview, setPreview] = useState({open: false, selectedTemplate: null});
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:68]: remove

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 2]
```diff
@@ -82,29 +65,20 @@ const FeedbackRequestPage = () => {
...
-      getMemberProfile();
-    }
-  }, [csrf, id]);
-
-  const hasFor = useCallback(() => {
-    return !id;
-  }, [id])
+  const handlePreviewClose = useCallback((selectedTemplate) => {
+    setPreview({open: false, selectedTemplate: selectedTemplate});
+  },[setPreview]);
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:72]: remove

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 3]
```diff
@@ -129,9 +103,14 @@ const FeedbackRequestPage = () => {
...
         return false;
     }
-  }, [activeStep, hasFor, hasTemplate, hasFrom, hasDue]);
+  }, [activeStep, hasTemplate, hasFrom, hasDue, hasFor]);
 
-  const handleSubmit = () => {};
+    if (activeStep < 1 || activeStep > steps.length) {
+
+    }
+
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:111]: remove

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 4]
```diff
@@ -172,30 +151,30 @@ const FeedbackRequestPage = () => {
...
-
   return (
     <div className="feedback-request-page">
+          {preview.selectedTemplate &&
+            <TemplatePreviewModal
+              template={preview.selectedTemplate}
+              open={preview.open}
+              onClose={handlePreviewClose}
+            />
+          }
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:162]: remove

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 5]
```diff
@@ -172,30 +151,30 @@ const FeedbackRequestPage = () => {
...
         <div>
-            <Button className={classes.actionButtons} onClick={onBackClick} disabled={activeStep <= 1}
-                    variant="contained">
-              Back
-            </Button>
-            <Button className={classes.actionButtons} onClick={onNextClick}
-                    variant="contained" disabled={!canProceed()} color="primary">
-              {activeStep === steps.length ? "Submit" : "Next"}
-            </Button>
+            <div>
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:166]: remove

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 6]
```diff
@@ -172,30 +151,30 @@ const FeedbackRequestPage = () => {
...
+              <Button className={classes.actionButtons} onClick={onBackClick} disabled={activeStep <= 1}
+                      variant="contained">
+                Back
+              </Button>
+              <Button className={classes.actionButtons} onClick={onNextClick}
+                      variant="contained" disabled={!canProceed()} color="primary">
+                {activeStep === steps.length ? "Submit" : "Next"}
+              </Button>
+            </div>
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:175]: remove

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 7]
```diff
@@ -7,32 +7,17 @@ import Button from "@material-ui/core/Button";
...
-  marginLeft: "2%",
-
-  },
-  stepContainer: {
-     ['@media min-width(321px) and (max-width:767px)']: { // eslint-disable-line no-useless-computed-key
-          width: '80%',
-        },
-         ['@media max-width(320px)']: { // eslint-disable-line no-useless-computed-key
-                 display: "none",
-                },
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:35]: put this back

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 8]
```diff
@@ -172,30 +151,30 @@ const FeedbackRequestPage = () => {
...
+              </Button>
+              <Button className={classes.actionButtons} onClick={onNextClick}
+                      variant="contained" disabled={!canProceed()} color="primary">
+                {activeStep === steps.length ? "Submit" : "Next"}
+              </Button>
+            </div>
           </div>
         </div>
-
-      <div className= {classes.stepContainer}>
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:198]: put back

File: web-ui/src/pages/FeedbackRequestPage.jsx [hunk 9]
```diff
@@ -207,14 +186,12 @@ const FeedbackRequestPage = () => {
             );
           })}
         </Stepper>
-        </div>
```
Comment (Approve212) [inline web-ui/src/pages/FeedbackRequestPage.jsx:210]: put back

**Review by mkimberlin — APPROVED** [review]

**Pull request merged**